### PR TITLE
fix #2811: Cart's Limited Supply status should not hide item title/quantity info 

### LINKED ISSUE #2811: Limited supply label not working correctly in cart.
### Expected behavior
When you have a limited supply label it should show up in the cart and not wipe out the items in cart quantity. It could show up in a similar place on the product image in cart as it does on the product grid. 

### Actual Behavior
Limited supply shows different language and removes the product info and quantity in the cart. 

### Steps to Reproduce the Behavior
Created a product with limited supply and put it in the cart. 

labels don t match up with limited supply

### PR DESCRIPTION
**Pull request by machikoyasuda**
- [x] Closes https://github.com/reactioncommerce/reaction/issues/2811
- [ ]: Note: This issue doesn't add a `# Left` badge. It just shows `Limited supply` if the Item/Variant has that on the Grid.
- [ ]: Note ii: This doesn't change the `Limited supply` badges that show up elsewhere, on the grid or the PDP.
- [x] Changed `Limited Supply` to `Limited supply`
- [x] Provide us detailed instructions on how we can test your PR:

1. Created a product with limited supply
2. Add that product to cart
3. Open cart
4. Confirm the item's title, variant title and quantity appears below the image in the cart
- [x] Design

before: No item title, variant title or quantity. Messed up badge.

screen shot 2017-09-20 at 11 03 17 am

after: brings back item title, etc. fixes badge.
screen shot 2017-09-29 at 10 55 34 am

### CODE REVIEW

**Comment by mikemurray** [comment]
Possibly missing limited supply badge when dealing with variant options.


basic_reaction_product

**Comment by rymorgan** [comment]
The limited supply label doesn't seem to cascade down the variant option. We're assuming that's always been the behavior. I'm making another issue to address that. So this one should be good to fix this bug.

File: imports/plugins/core/checkout/client/components/cartItems.js [hunk 1]
```diff
@@ -30,7 +31,16 @@ class CartItems extends Component {
...
     return (
-      <div className="cart-items" key={item._id} style={{ display: "inline-block" }}>
+      <div
+        className="cart-items"
+        key={item._id}
+        style={{ display: "inline-block" }}
+      >
+        {handleLowInventory(item) &&
+          <div className="badge badge-top badge-low-inv-warning">
+            { i18next.t("cartDrawerItems.limitedSupply", { default: "Limited Supply" }) }
```
Comment (mikemurray) [inline imports/plugins/core/checkout/client/components/cartItems.js]: Prefer usage of translation component, over importing `i18next` where possible:

```
<Components.Translation i18nKey="cartDrawerItems.limitedSupply" defaultValue="Limited Supply" />
```

**Comment by spencern** [comment]
@machikoyasuda you've got pending request changes here

**Pull request merged**